src/jit/lowerxarch.cpp

```diff
@@ -903,6 +903,16 @@ void Lowering::TreeNodeInfoInit(GenTree* stmt)
             }
 
             // Set destination candidates for return value of the call.
+#ifdef _TARGET_X86_
+            if ((tree->gtCall.gtCallType == CT_HELPER) && (tree->gtCall.gtCallMethHnd == compiler->eeFindHelper(CORINFO_HELP_INIT_PINVOKE_FRAME)))
```

Comment:
I see this condition is used more than once (also in codegenxarch.cpp and in morph.cpp).  I would suggest to abstract it in a member routine of call node

e.g.
bool GenTreeCall::IsInitPInvokeFrameHelper() const